src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -440,4 +439,63 @@ public String convert(final String name) {
 		});
 	}
 	
+	@Override
+	public List<String> onTabComplete(CommandSender sender, Command cmd, String s, String[] args) {
+		File file = new File(Skript.getInstance().getDataFolder(), "scripts");
+		File[] listFiles = file.listFiles();
+		if (listFiles != null) {
+			List<File> files = new ArrayList<>(Arrays.asList(listFiles));
+			List<String> command = new ArrayList<>(Arrays.asList("reload", "enable", "disable", "update", "help", "info"));
+			List<String> reload = new ArrayList<>(Arrays.asList("@all", "@scripts", "@config", "@aliases"));
+			List<String> enable = new ArrayList<>(Collections.singletonList("@all"));
+			List<String> disable = new ArrayList<>(Collections.singletonList("@all"));
+			List<String> backup = new ArrayList<>(Collections.singletonList("@all"));
+			List<String> update = new ArrayList<>(Arrays.asList("check", "changes", "download"));
+			List<String> list = new ArrayList<>();
+			if (!sender.isOp()) {

```

Comment:
Do we want to be doing an isOp check for this? We should check the sender’s permissions, no?